### DISCUSSION TITLE
set the current inverter limit via a physical button

### DISCUSSION THREAD

**Discussion by AIMutant** [tbnobody/OpenDTU#1389] [Ideas]
Just as a brainstorming idea, is it possible to reduce the inverter limit to a predefined value by pressing a physical button?
I thought it might be possible to connect a button to the ESP32 pins and then on a press, send a command to reduce the limit to value X.

Whould this be achievable, or would it be better to do it via mqtt?

**Comment by DPO99** [comment]
I think it is possible but i dont think that is an idea for the basic firmware. You can create a PullRequest and maybe someone can solve it. Via MQTT and additional hardware there are many ways to build what you want.
